### PR TITLE
Finish slices guide

### PR DESCRIPTION
**Pull request by timriley**
Add sections on standard app components, slice settings and slice loading.

### CODE REVIEW

**Comment by timriley** [comment]
[Slices: per slice settings](https://trello.com/c/MNrvqdMF/321-slices-per-slice-settings)

**Comment by timriley** [comment]
[Slices: HANAMI_SLICES env var](https://trello.com/c/U0HH9AyU/347-slices-hanamislices-env-var)

File: content/v2.0/architecture/slices.md [hunk 1]
```diff
@@ -258,7 +282,76 @@ module CDN
...
+module CDN
+  class Settings < Hanami::Settings
+    setting :cdn_api_key, Types::String
+  end
+end
+```
+
+With this in place, the `"settings"` component within your slice will be an instance of this slice-specific settings object.
+
+```
```
Comment (andrewcroome) [inline content/v2.0/architecture/slices.md:303]:
```suggestion
```ruby
```

File: content/v2.0/architecture/slices.md [hunk 2]
```diff
@@ -258,7 +282,76 @@ module CDN
...
+
+## Slice loading
+
+Hanami will load all slices when your app boots. However, for certain workloads of your app, you may elect to load only a specified list of slices.
+
+Loading specific slices brings the benefit of stronger code isolation, faster boot time and reduced memory usage. If your app had a background worker that processed jobs from one slice only, then it would make sense to load only that slice for the worker's process.
+
+To do this, set the `HANAMI_SLICES` environment variable with a comma-separated list of slice names.
+
+```ruby
```
Comment (andrewcroome) [inline content/v2.0/architecture/slices.md:339]: Looks like $ gets a bit weird if the code is flagged as Ruby. Change this one to shell?

Screen Shot 2022-11-21 at 12 11 40 pm

**Review by andrewcroome — APPROVED** [review]
Thanks @timriley! Two minor suggestions.

File: content/v2.0/architecture/slices.md (hunk 2 above)
Comment (timriley) [reply content/v2.0/architecture/slices.md:339]: Oh yeah, thanks, that was an oversight!

**Pull request merged**